x-pack/plugins/alerting_builtins/server/alert_types/index_threshold/routes/indices.ts

```diff
@@ -137,3 +140,9 @@ async function getAliasesFromPattern(
 
   return result;
 }
+
```

Comment:
I like having these little stand-alone interfaces like this, for cases like this.